StoreKitUnitTests/ProductDetailsTests.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ProductDetailsTests.swift
+//
+//  Created by Andrés Boedo on 1/9/21.
+
+import Nimble
+@testable import RevenueCat
+import StoreKitTest
+import XCTest
+
+class ProductDetailsTests: XCTestCase {
+
+    var testSession: SKTestSession!
+    var userDefaults: UserDefaults!
+
+    override func setUpWithError() throws {
+        testSession = try SKTestSession(configurationFileNamed: "UnitTestsConfiguration")
+        testSession.disableDialogs = true
+        testSession.clearTransactions()
+    }
+
+    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    func testSK1AndSK2DetailsAreEquivalent() async throws {
+        guard #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *) else {
+            throw XCTSkip("Required API is not available for this test.")
```

Comment:
replaced early returns with these, which will show the test as skipped for older OS versions (still counts as all passed, but will it'll say something like "executed 832 tests, with 0 failures and 3 skipped"

Reply:
Oh that's great, I will add it to my PR